auth/providers/azure/pop_tokenverifier.go

```diff
@@ -125,7 +125,7 @@ func (p *PoPTokenVerifier) ValidatePopToken(token string) (string, error) {
 			if klog.V(6).Enabled() {
 				klog.V(6).Infof("pop token validation running with hostName: %s. Request is coming for hostName: %s", p.hostName, reqHostName)
 			}
-			if !strings.EqualFold(reqHostName, p.hostName) {
+			if !strings.HasSuffix(reqHostName, p.hostName) {
```

Comment:
This is needed while enabling PoP. Is it related to obo?

Reply:
no but since we had identified it while discussing thought to take up the change

Reply:
removed